File: test/fixtures/extractor-repo/src/dotnet/PanelService.cs
using System;
using Acme.Contracts;

namespace Acme.Panels;

public record Panel(string Id);

public class PanelService
{
    private readonly PanelClient client;

    public PanelService(PanelClient client)
    {
        this.client = client;
    }

    public Panel LoadPanel(string id)
    {
        return client.Fetch(new Uri($"/panels/{id}", UriKind.Relative));
    }

    private string Normalize(string id) => id.Trim();

    public string Name { get; init; } = "panels";
}

File: test/fixtures/extractor-repo/src/ruby/panel_service.rb
require "json"
require_relative "panel_client"

module Panels
  DEFAULT_PANEL = "panel"

  class PanelService
    def load_panel(id)
      PanelClient.new.fetch(id)
    end

    def self.audit(event)
      JSON.generate(event)
    end
  end
end
